### ISSUE TITLE
Release option (debians)

### ISSUE THREAD

**Issue by reinzor** [Rayman/ros-get#75]
What about generating a fat debian using:

https://gitlab.com/VictorLamoine/bloom_local_release

**Comment by Rayman** [comment]
Cool project!

So what are the commands that it acually runs? These 2?
```bash
bloom-generate rosdebian --os-name $os --os-version $os_version --ros-distro $ros_distro
fakeroot debian/rules binary
```

**Comment by reinzor** [comment]
Yes, that works. I tested this. However, you still need to setup rosdeps since `bloom-generate` will complain about this.

**Comment by reinzor** [comment]
For installing multiple deb files respecting dependencies:

https://askubuntu.com/questions/708067/install-multiple-deb-files-while-respecting-their-dependencies
